rele/config.py

```diff
@@ -83,6 +83,20 @@ def setup(setting=None, **kwargs):
     return config
 
 
+def subscriptions_have_same_memory_address(a_subscription, another_subscription):
+    return id(a_subscription) == id(another_subscription)
+
+
+def subscription_already_registered(subscriptions, current_subscription):
```

Comment:
```suggestion
def is_subscription_already_registered(subscriptions, current_subscription):
```
I would include the `is` verb in the method naming